### PR TITLE
chore(deps): bump net package version

### PR DESCRIPTION
**Pull request by rogercoll**

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -9,4 +9,6 @@ COPY tools /app/tools
 
 COPY go.mod go.sum Makefile /app/
 
+ENV GOFLAGS="-buildvcs=false"
```
Comment (rubenruizdegauna) [inline build/Dockerfile:12]: is this necessary ☝️ ?  (just ensuring is not a leftover from local)

Comment (rogercoll) [reply build/Dockerfile:12]: Yes, is the same issue that happened with the agent a few weeks ago: https://github.com/newrelic/nri-flex/actions/runs/4404183693/jobs/7713478560#step:4:184

Comment (rogercoll) [reply build/Dockerfile:12]: Goreleaser fails with new Go container image due VCS and project's name. (we need to bump GoReleaser version too)

**Review by rubenruizdegauna — APPROVED** [review]
LGTM 🔥

**Pull request merged**